### DISCUSSION TITLE
Браузер от GitHub

### DISCUSSION THREAD

**Discussion by OKRUgit** [community/community#77241] [Discussions] [answered]
### Select Topic Area

Question

### Body

Система linux Debian 10 окружение KDE Последняя версия. домашний ПК, не сматр и прочие  гаджеты.
Сегодня нажал на кнопку в браузере хрома и там сказано переход на "приложение", нажал.
Что увидел, все сделано не плохо, не отвлекает и мешает реклама. Сразу думаю надо на GitHub так всегда заходить, понравилось!
Однако не смог в системе сделать значек для запуска сразу в этом приложении, не понял как его сделать, попробовал поискать и ничего не получилось.
Если есть такая возможность то где эти настройки, если нет, Былобы неплохо сделть . Как пока первые пробы тут много ещё не знаю Подсказывайте. пожалуйста.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @OKRUgit , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
